ExampleApp/AppDelegate.swift

```diff
@@ -211,6 +216,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         }
     }
+
+    private func handleExampleVerify1FA(verify1FACallbackUrl: URL, host: String) {
+        let VERIFY_1FA = "complete1FA"
+        let YAHOO_HOST = "completeYahoo1FA"
```

Comment:
Why are there Yahoo changes as part of this?

Reply:
Yahoo has his own launcher (line 240).

Reply:
No, I don't think so. I implemented the full feature because the trade service returns now a "complete1FA" closure.
In that case, we could ignore and do nothing here or return an error ? : https://github.com/tradingticket/TradeItIosTicketSDK2/blob/1286aedc30d482e0ba61ab777fefa3a6dd375a55/TradeItIosTicketSDK2/TradeItYahooTradePreviewViewController.swift#L113